Lib/fontTools/subset/__init__.py

```diff
@@ -2207,28 +2207,58 @@ def prune_pre_subset(self, font, options):
 @_add_method(ttLib.getTableClass('cmap'))
 def subset_glyphs(self, s):
 	s.glyphs = None # We use s.glyphs_requested and s.unicodes_requested only
+
+	tables_format12 = []
+	table_plat0_enc3 = {}
+	table_plat3_enc1 = {}
+
 	for t in self.tables:
+		if t.platformID == 0 and t.platEncID == 3:
+			table_plat0_enc3[t.language] = t
+		if t.platformID == 3 and t.platEncID == 1:
+			table_plat3_enc1[t.language] = t
+
 		if t.format == 14:
 			# TODO(behdad) We drop all the default-UVS mappings
 			# for glyphs_requested.  So it's the caller's responsibility to make
 			# sure those are included.
 			t.uvsDict = {v:[(u,g) for u,g in l
-					      if g in s.glyphs_requested or u in s.unicodes_requested]
-				     for v,l in t.uvsDict.items()}
+						  if g in s.glyphs_requested or u in s.unicodes_requested]
+					 for v,l in t.uvsDict.items()}
```

Comment:
is the whitespace change only cosmetic? (one day we'll blacken everything so I wouldn't bother)

Reply:
Removed, that was unintentional.